lib/Inst/Inst.cpp

```diff
@@ -574,6 +578,7 @@ Inst *InstContext::createVar(unsigned Width, llvm::StringRef Name,
   I->Number = Number;
   I->Width = Width;
   I->Name = Name;
+  I->Range = Range;
```

Comment:
I think you should add some assertions here, making sure that the widths of Range, KnownZeros, and KnownOnes match the width of the var, to make sure we're never getting stuck using the default values for these parameters